nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-components/src/main/java/org/apache/nifi/controller/service/ServiceStateTransition.java

```diff
@@ -76,7 +76,7 @@ public boolean enable(final ControllerServiceReference controllerServiceReferenc
             }
 
             state = ControllerServiceState.ENABLED;
-            logger.debug("{} transitioned to ENABLED", controllerServiceNode);
+            logger.info("{} is now fully ENABLED", controllerServiceNode);
```

Comment:
This change from `debug` to `info` results in two similar log lines at the `INFO` level:

```
INFO [Timer-Driven Process Thread-4] o.a.n.c.service.ServiceStateTransition StandardControllerServiceNode[service=StandardHttpContextMap[id=01821002-31b4-1869-4f8e-0f3141d6c1c2], name=StandardHttpContextMap, active=true] is now fully ENABLED
INFO [Timer-Driven Process Thread-4] o.a.n.c.s.StandardControllerServiceNode Successfully enabled StandardControllerServiceNode[service=StandardHttpContextMap[id=01821002-31b4-1869-4f8e-0f3141d6c1c2], name=StandardHttpContextMap, active=true]
```

Although these logs come from different locations, they seem duplicative to a general user. Should the change to `info` be reverted, or should the other message level be changed?